schema/reportdb/uyuni-reportdb-schema.spec

```diff
@@ -0,0 +1,98 @@
+#
+# spec file for package susemanager-reportdb-schema
+#
+# Copyright (c) 2021 SUSE LLC
+# Copyright (c) 2008-2018 Red Hat, Inc.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+%{!?fedora: %global sbinpath /sbin}%{?fedora: %global sbinpath %{_sbindir}}
+
+Name:           uyuni-reportdb-schema
+Summary:        Report DB SQL schema for Spacewalk server
+License:        GPL-2.0-only
+Group:          Applications/Internet
+
+Version:        4.3.4
+Release:        1
+Source0:        %{name}-%{version}.tar.gz
+Source1:        %{name}-rpmlintrc
+
+URL:            https://github.com/uyuni-project/uyuni
+BuildArch:      noarch
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+
+Requires:       susemanager-schema-core
+Requires:       susemanager-schema-sanity
+
+Provides:       uyuni-reportdb-schema = %{version}
```

Comment:
I think we do not need this Provides as this is the default provides added by the name of the package